src/group_theory/group_action.lean

```diff
@@ -28,6 +28,9 @@ theorem mul_smul (a₁ a₂ : α) (b : β) : (a₁ * a₂) • b = a₁ • a₂
 
 lemma smul_smul (a₁ a₂ : α) (b : β) : a₁ • a₂ • b = (a₁ * a₂) • b := (mul_smul _ _ _).symm
 
+lemma smul_comm {α : Type u} {β : Type v} [comm_monoid α] [mul_action α β] (a₁ a₂ : α) (b : β) :
```

Comment:
Should this be something like `smul_left_comm`? `smul_right_comm` would only make sense when an object acts on itself, so maybe that's an argument to stick with the shorter name.

Reply:
There's certainly a case for `smul_left_comm` but I'm inclined to prefer `smul_comm` for the reason you state (right case requires self-action). Happy to change if you feel differently of course!